### PR TITLE
OSCAL Tools: Correct typo for RegScale entry + link to license

### PR DESCRIPTION
**Pull request by akarmel**
Corrected typo in "Description" field with caused content to wrap into the "Type" field.   Included a link to the Community Edition License.

### CODE REVIEW

File: docs/content/tools/_index.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Valid OSCAL is open-ended in application. Some of the tools described provide va
...
+| [RegScale (formerly Atlasity)](https://regscale.com) | RegScale (spun out of C2 Labs)  | RegScale Community Edition is a free to use, API-centric real-time Governance, Risk and Compliance (GRC) platform that deploys in any environment, shifting left compliance by integrating with security and compliance tools via API to keep compliance documentation continuously up to date. GRC staff can work in the UI, engineers can write scripts to feed the API, and both human- and machine-readable (OSCAL - all layers supported) content is automatically generated on demand, achieving a continuous Authority to Operate (cATO). Additional information can be found here: [RegScale Delivers Free Platform to Create OSCAL Content](https://regscale.com/oscal). | [Community Edition](https://qa-ce.regscale.com/assets/eula-ce.pdf) |
```
Comment (bradh) [inline docs/content/tools/_index.md:60]: The list should be in alphabetical order.

I also agree that "Community License" is a potentially confusing term. It has some pretty non-community terms. IMO its a different Commercial License. A middle way forward would be to do it the way Xacta is done:

```suggestion
| [RegScale (formerly Atlasity)](https://regscale.com) | RegScale (spun out of C2 Labs)  | RegScale Community Edition is a free to use, API-centric real-time Governance, Risk and Compliance (GRC) platform that deploys in any environment, shifting left compliance by integrating with security and compliance tools via API to keep compliance documentation continuously up to date. GRC staff can work in the UI, engineers can write scripts to feed the API, and both human- and machine-readable (OSCAL - all layers supported) content is automatically generated on demand, achieving a continuous Authority to Operate (cATO). Additional information can be found here: [RegScale Delivers Free Platform to Create OSCAL Content](https://regscale.com/oscal). | [license](https://qa-ce.regscale.com/assets/eula-ce.pdf) |
```

Comment (david-waltermire-nist) [reply docs/content/tools/_index.md]: @akarmel I agree with the changes by @bradh. Please refine this entry.

**Review by david-waltermire-nist — CHANGES_REQUESTED** [review]
@akarmel This list is in alphabetical order. Please adjust the order of regscale to come before `Xacta 360`. Thanks!

**Review by david-waltermire-nist — APPROVED** [review]
I updated the PR to address the ordering.

**Pull request merged**